### DISCUSSION TITLE
unstable mode update, whats new?

### DISCUSSION THREAD

**Discussion by ghost** [Monika-After-Story/MonikaModDev#8005] [QA - tech support + general]
so, i just updated to unstable (basically a test update for beta MAS or something). usually, there is new stuff but I don't see anything different. can someone tell me whats different for the unstable update?

**Comment by Booplicate** [comment]
Patch notes for unstable releases are being posted in [the community discord server](https://discord.gg/K2KuJeX).

**Comment by ghost** [comment]
discord is blocked for me, can u send a screenshot of the patch notes?
